### PR TITLE
refactor: move install command into local setup/upgrade

### PR DESCRIPTION
**Pull request by bohdand-weka**

### CODE REVIEW

File: internal/cli/local/dump/cli.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	Cli    hooks.Cli
+	config dump.Config
+)
+
+func init() {
+	Cli.AddHook(func(appCmd *cobra.Command) {
+		appCmd.AddCommand(dumpCmd)
+
+		dumpCmd.Flags().BoolVarP(&config.Verbose, "verbose", "v", false, "Increase verbosity to display debug information during collection phase.")
+		dumpCmd.Flags().BoolVar(&config.InclideSensitive, "include-sensitive", false, "Include sensitive data in the archive (e.g., values overrides). Use with caution.")
```
Comment (mfuhol-weka) [inline internal/cli/local/dump/cli.go]: config.InclideSensitive

File: internal/local/dump/dump.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	if err != nil {
+		return err
+	}
+	stderr, err := shell.StderrPipe()
+	if err != nil {
+		return err
+	}
+
+	infoLogWriter := utils.NewWriteScanner(func(b []byte) {
+		fmt.Println(string(b))
```
Comment (mfuhol-weka) [inline internal/local/dump/dump.go]: maybe replace with logger?

File: internal/cli/local/setup/run.go [hunk 3]
```diff
@@ -58,47 +60,69 @@ func readConfiguration(jsonConfig string) (*chart.Configuration, error) {
 	return config, nil
 }
 
-func runInstallOrUpgrade(cmd *cobra.Command, args []string) error {
-	if installCmdOpts.remoteVersion != "" && !installCmdOpts.remoteDownload {
-		return fmt.Errorf("%w: --remote-version can only be used with --remote-download", utils.ErrValidationFailed)
+func runSetup(cmd *cobra.Command, args []string) error {
```
Comment (mfuhol-weka) [inline internal/cli/local/setup/run.go:63]: I'm confused why you are making overlay on top of chart installer and not just making another standalone command which depends on k3s/chart installer utilities

Comment (bohdand-weka) [reply internal/cli/local/setup/run.go:63]: you wanted to keep old commands? I think main point Anton was aksing was to have single command for all install, and single command for upgrade

Comment (mfuhol-weka) [reply internal/cli/local/setup/run.go:63]: I thought just to make a low-level set of commands and high-level, but if you want to merge them up - okay

Comment (bohdand-weka) [reply internal/cli/local/setup/run.go:63]: I think about adding sub-command later like install just new chart, but it needs to be discussed how we see it

**Review by gonenidan — APPROVED** [review]

**Pull request merged**